### ISSUE TITLE
EVPath: Add configuration options for additional features

### ISSUE THREAD

**Issue by wspear** [spack/spack#12963]
We are creating a package for a program that depends on EVPath but it needs EVPath's atl, ffs, dill, enet (currently provided) and cercs_env components. Could we get these options added to the EVPath package?

**Comment by adamjstewart** [comment]
Hi @wspear, you can add whatever variants you want to the `libevpath` package! PRs are always welcome. I just checked and that package currently doesn't list any `maintainers`, so technically no one is in charge of it.

**Comment by alalazo** [comment]
@wspear I updated the tags as usually we use `feature` for core Spack features. Feel free to change anything if what I added seems not appropriate to you.

**Comment by chuckatkins** [comment]
@eisenhauer I suspect it would be trivial to add the evpath dependencies as packages, yes?

**Comment by eisenhauer** [comment]
Yes, presumably....   (cercs_env is no longer required by anything current.)
